### ISSUE TITLE
Add a custom next and previous button

### ISSUE THREAD

**Issue by dmorda** [FormidableLabs/nuka-carousel#630]
# Bugs and Questions

### Describe Your Environment

- What version of nuka-carousel are you using?
4.6.0

- What version of React are you using?
16.12.0

- What browser are you using?
Chrome 79

- What machine are you on?
Macbook Air

### Describe the Problem
I've implemented the carousel and done some basic styling. One thing I'm stuck on is being able to disable the built in controls and defining my own that I can use to control the next and previous navigation.

Basically, I want to have something like this sitting right below the carousel that allows me to navigation the slides and I can control the styling of it.

[PREVIOUS] 1 of 3 [NEXT]

Is there an easy way to do this or could you provide me a little guidance on the best way to go about accomplishing this?

Thanks in advance!

**Comment by damir-sirola** [comment]
You can implement custom next and previous buttons by using render controls. Take a look at [https://github.com/FormidableLabs/nuka-carousel#rendercontrols](https://github.com/FormidableLabs/nuka-carousel#rendercontrols)

**Comment by sarmeyer** [comment]
Added this functionality in #644, will be released soon!
